src/extension/gogoduck/doc/wps-acorn.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Fair enough, just thought it might come handy. I understand what you're saying. How should I arrange it then?